bindata/v4.1.0/kube-controller-manager/csr_approver_serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: cluster-csr-approver-controller
```

Comment:
I'm a little surprised this is required instead of auto-created, but ok.

Reply:
hmmmm just to educate me - would you mind to explain how this auto-generate mechanism works and where I can find it? Maybe I implement this somehow differently?

Reply:
I think the ClientBuilder you're using in the CPO PR is capable of creating the SA

Reply:
Let me verify that by creating a new cluster without this commit. Will let you know how did it go.

Reply:
Yes, you're absolutely correct! Although the Service Account is created in `openshift-infra` namespace, which is quite surprising (it thought it would be created in `openshift-kube-controller-manager`, where the controller runs).

Closing this PR.